### PR TITLE
Maintenance and change newGCMS format

### PR DESCRIPTION
**Pull request by maasencioh**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
-  - "4"
-  - "stable"
-sudo: false
+  - 8
+  - 6
```
Comment (targos) [inline .travis.yml]: can you keep node 4 compatibility? I don't want to have to release a major of this package

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
-# JCAMP converter
+# jcampconverter
```
Comment (targos) [inline README.md]: why change the name?

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Luc Patiny
+Copyright (c) 2017 cheminfo
```
Comment (targos) [inline LICENSE]: original copyright must be kept

File: .travis.yml (hunk 1 above)
Comment (maasencioh) [inline .travis.yml]: but changing the return value in newGCMS is already a major

Comment (targos) [inline .travis.yml]: yeah but we are the only users of this API

File: README.md (hunk 2 above)
Comment (maasencioh) [inline README.md]: it is the standard for the generator and the name of the library, but I don't mind to change it back

File: src/index.js [hunk 4]
```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+const defaultOptions = {
```
Comment (targos) [inline src/index.js]: you have to put this in the getConverter function for web worker compatibility

Comment (maasencioh) [inline src/index.js]: didn't know that, thanks

File: src/index.js [hunk 5]
```diff
@@ -855,6 +859,7 @@ function postToWorker(input, options) {
     });
 }
 
+/* eslint-disable no-undef */
```
Comment (targos) [inline src/index.js]: Is that because of the browser globals?
The solution is to add `browser: true` to the ESLint environment

Comment (maasencioh) [inline src/index.js]: cool! it was for that

**Comment by maasencioh** [comment]
@targos do you agree?

**Comment by targos** [comment]
if there is no old format anymore, do we still need the option?

**Comment by maasencioh** [comment]
@targos yes if the format has this structure:
```
##TITLE=TEST
##JCAMP-DX=4.24
##DATA TYPE=MASS SPECTRUM
##XUNITS=M/Z
##YUNITS=RELATIVE ABUNDANCE
##FIRSTX=4.0058
##LASTX=34.2505
##FIRSTY=248.7503
##LASTY=9931184
##NPOINTS=10530
##PEAK TABLE=(XY..XY)
4.0058	16811.58
4.00867	837.751
4.01153	103.5574
4.01438	0
``` 
 we could force it to be a 1D serie as:
```js
{
     time: [...],
     series: {
          intensity: {...}
     }
}
```

**Pull request merged**